packages/fluent-theme/src/fluent/styles/TeachingBubble.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { Depths } from '../FluentDepths';
+import { FontSizes } from '../FluentType';
+import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { ITeachingBubbleStyleProps, ITeachingBubbleStyles } from 'office-ui-fabric-react/lib/TeachingBubble';
+
+export const TeachingBubbleStyles = (props: ITeachingBubbleStyleProps): Partial<ITeachingBubbleStyles> => {
+  const { hasCondensedHeadline, hasSmallHeadline, theme } = props;
+  const { palette } = theme;
+
+  let headlineSize = FontSizes.size14;
+  if (!hasCondensedHeadline && !hasSmallHeadline) {
+    headlineSize = FontSizes.size20;
+  }
+
+  return {
+    headline: {
+      fontSize: headlineSize,
+      fontWeight: FontWeights.semibold
+    },
+    footer: {
+      selectors: {
+        '.ms-Button:not(:first-child)': {
+          marginLeft: '16px'
+        }
+      }
+    },
+    closeButton: {
+      backgroundColor: 'transparent',
+      selectors: {
+        '&:hover': {
+          color: palette.black
+        }
+      }
+    },
+    subComponentStyles: {
+      callout: {
+        root: {
+          boxShadow: Depths.depth16
```

Comment:
Looks weird to me that before this was:
https://github.com/OfficeDev/office-ui-fabric-react/blob/5bc73a304b843bf183bf77659e0165d395f4af91/packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts#L91
and now from nowhere `depth16`, but designers know better 🙄 .